internal/monitors/kubernetes/cluster/metrics/cache.go

```diff
@@ -161,6 +161,14 @@ func (dc *DatapointCache) HandleAdd(newObj runtime.Object) interface{} {
 		dps = datapointsForCronJob(o)
 		dimProps = dimPropsForCronJob(o)
 		kind = "CronJob"
+	case *v1.PersistentVolume:
```

Comment:
I'm pretty sure you have to start syncing them in the `clusterstate.go` module for this to ever get triggered.